packages/pwa/README.md

```diff
@@ -31,18 +31,18 @@ See the [Localization README.md](./app/translations/README.md) for important set
 
 ## URL Customization
 
-You can customize how storefront URLs are formatted in the `pwa-kit-config.json` file.
+You can customize how storefront URLs are formatted in the you applications configuration file.
 
 ```json
```

Comment:
```suggestion
``` jsonc
```

Reply:
TIL what jsonc was. But actually, I just realized that the config is a js file now.. So I've update that.